src/Pages/CatPresentation/index.jsx

```diff
@@ -135,6 +143,10 @@ function CatPresentationPage() {
             name={item.name}
           />
         ))}
+        {!isFetching && !isFetchingNextPage && !search && (
+          <Waypoint onEnter={handleOnNextClick} onLeave={handleOnReturnClick} />
```

Comment:
```suggestion
          <Waypoint onEnter={handleOnNextClick} />
```

we don't need `onLeave` here because we don't need to fetch a previous page.

Reply:
checked it